### PR TITLE
Upgrade libwebrtc to 108.

### PR DESCRIPTION
**Pull request by jianjunz**
This PR only enables OWT build with clang after upgrading libwebrtc to 108. Not all features are working properly.

This PR depends on https://github.com/open-webrtc-toolkit/owt-deps-webrtc/pull/162.

### CODE REVIEW

File: talk/owt/sdk/base/mediautils.cc [hunk 1]
```diff
@@ -99,27 +99,7 @@ absl::optional<unsigned int> MediaUtils::GetH264TemporalLayers() {
...
-      break;
-  }
-  if (prefix_state.has_value()) {
-    temporal_id = prefix_state.value().temporal_id;
-    is_idr = prefix_state.value().idr_flag == 1 ? true : false;
-    priority_id = prefix_state.value().priority_id;
-    return true;
-  }
-
+  // TODO: PrefixParser does not exist. Temporal scalability is disabled.
```
Comment (taste1981) [inline talk/owt/sdk/base/mediautils.cc:102]: Prefix parser is added by us. Make sure you add it back.

File: talk/owt/sdk/base/peerconnectiondependencyfactory.cc [hunk 2]
```diff
@@ -265,63 +260,36 @@ PeerConnectionDependencyFactory::CreatePeerConnectionOnCurrentThread(
...
-  }
-  if (ice_port_ranges.data.min > 0 &&
-      ice_port_ranges.data.max > ice_port_ranges.data.min) {
-    port_allocator->SetDataPortRange(ice_port_ranges.data.min,
-                                     ice_port_ranges.data.max);
-  }
   return (pc_factory_->CreatePeerConnection(config, std::move(port_allocator),
-                                            nullptr, observer))
-      .get();
+                                            nullptr, observer));
```
Comment (taste1981) [inline talk/owt/sdk/base/peerconnectiondependencyfactory.cc:264]: Where do we specify the port ranges?

Comment (jianjunz) [reply talk/owt/sdk/base/peerconnectiondependencyfactory.cc:264]: The port range API is still in GlobalConfiguration, but it's not passed to port allocator. Created an issue for tracking https://github.com/open-webrtc-toolkit/owt-client-native/issues/619.

File: talk/owt/sdk/base/mediautils.cc (hunk 1 above)
Comment (jianjunz) [reply talk/owt/sdk/base/mediautils.cc:102]: Tracked by #618.

**Review by taste1981 — APPROVED** [review]

**Pull request merged**